applications/StructuralMechanicsApplication/custom_python/structural_mechanics_python_application.cpp

```diff
@@ -120,6 +120,11 @@ BOOST_PYTHON_MODULE(KratosStructuralMechanicsApplication)
     KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS( NODAL_STIFFNESS )
     KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS( NODAL_DAMPING_RATIO )
 
+    // For explicit central difference scheme
+    KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(MIDDLE_VELOCITY)
```

Comment:
One question, do you really need to store all that data?, there is any explicit computation that would allow you to skip to store that  info?

Reply:
I am using central differences so I need to store the middle velocity for each node. And central differences is (to my knowledge) the best to choose

Reply:
i am not into the details here...but the less database & memory the better...